ui/src/AutomationService/GroupView.purs

```diff
@@ -1,94 +1,22 @@
 module AutomationService.GroupView
```

Comment:
This file is still a mess, but we'll deal with that when the Group data is where we want it.